userspace/libsinsp/container_engine/docker_common.cpp

```diff
@@ -698,6 +698,10 @@ bool docker_async_source::parse_docker(const docker_async_instruction& instructi
 		container.m_is_pod_sandbox = true;
 	}
 
+	// Get the created time - this will be string format i.e. "YYYY-MM-DDTHH:MM:SS"
+	// Convert it to seconds.
+	container.m_created_time = static_cast<int64_t>(get_epoch_utc_seconds(root["Created"].asString()));
```

Comment:
from a quick grep, the default format used by `get_epoch_utc_seconds` is:

```
time_t get_epoch_utc_seconds(const std::string& time_str, const std::string& fmt = "%Y-%m-%dT%H:%M:%SZ");
```

I understand you tested it and it works with the missing Z?

Reply:
It was my comment that was wrong. 

```
kramkumar@debianDevEnv:~/kube-agentConfigFiles/officialCloudScripts$ docker inspect 5514e1776106 | grep -i created
        "Created": "2020-06-03T23:08:40.05444664Z",
``` 
the original time format in `docker inspect` has the Z

Reply:
Fixed the comment.